admin/views/tabs/dashboard/integrations.php

```diff
@@ -1,50 +0,0 @@
-<?php
-/**
- * WPSEO plugin file.
- *
- * @package WPSEO\Admin\Views
- *
- * @uses Yoast_Form $yform Form object.
- */
-
-use Yoast\WP\SEO\Presenters\Admin\Alert_Presenter;
-
-if ( ! defined( 'WPSEO_VERSION' ) ) {
-	header( 'Status: 403 Forbidden' );
-	header( 'HTTP/1.1 403 Forbidden' );
-	exit();
-}
-
-$integration_toggles = Yoast_Integration_Toggles::instance()->get_all();
```

Comment:
It's maybe also worth checking `src/integrations/third-party/wincher.php` and `src/integrations/third-party/wordproof-integration-toggle.php`: there are two filters there adding toggles in the Integrations page.

Reply:
They are still needed for the multisite admin too